Makefile

```
@@ -20,8 +20,8 @@ VALEOPTS        ?=
 # Internal variables.
 SPHINXBUILD     = "$(realpath bin/sphinx-build)"
 SPHINXAUTOBUILD = "$(realpath bin/sphinx-autobuild)"
-DOCS_DIR        = ./docs/source/
-BUILDDIR        = ../_build/
+DOCS_DIR        = ./docs/
+BUILDDIR        = ./_build/
```

Revert. The make command is issued at the root of the repo, and that's where I expect to see an ignored `_build` directory, so I don't have to go up and down the hierarchy.

```suggestion
BUILDDIR        = ../_build/
```

---

ok, in the old one, the build stays in `docs`, that's why I did it.

---

Yeah, the original config was wrong, too (should have been `../../_build`), but in the new context, it is now correct.